src/routes/SearchResult.js

```diff
@@ -4,9 +4,253 @@ import styles from '../views/SearchResult.module.css';
 class SearchResult extends Component {
     render() {
         return (
-            // type your html here
-            // styling is in views/SearchResult.module.css
-            <h1>Hello World</h1>
+            // typed my html here
```

Comment:
Comments inside the React language doesn't work like this. It is causing errors. Comment should be like this `{/* typed my html here */}`